src/settings/languages.js

```diff
@@ -33,41 +34,42 @@ export type Language = {|
-  { locale: 'hu', name: 'Hungarian', nativeName: 'Magyar' },
-  { locale: 'id', name: 'Indonesian', nativeName: 'Bahasa Indonesia' },
-  { locale: 'it', name: 'Italian', nativeName: 'Italiano' },
-  { locale: 'ja', name: 'Japanese', nativeName: '日本語' },
-  { locale: 'ko', name: 'Korean', nativeName: '한국어' },
-  { locale: 'lt', name: 'Lithuanian', nativeName: 'Lietuvių' },
-  { locale: 'ml', name: 'Malayalam', nativeName: 'മലയാളം' },
-  { locale: 'fa', name: 'Persian', nativeName: 'فارسی' },
-  { locale: 'pl', name: 'Polish', nativeName: 'Polski' },
-  { locale: 'pt', name: 'Portuguese', nativeName: 'Português' },
-  { locale: 'pt_PT', name: 'Portuguese (Portugal)', nativeName: 'Português (Portugal)' },
-  { locale: 'ro', name: 'Romanian', nativeName: 'Română' },
-  { locale: 'ru', name: 'Russian', nativeName: 'Русский' },
-  { locale: 'sr', name: 'Serbian', nativeName: 'Српски' },
-  { locale: 'es', name: 'Spanish', nativeName: 'Español' },
-  { locale: 'sv', name: 'Swedish', nativeName: 'Svenska' },
-  { locale: 'ta', name: 'Tamil', nativeName: 'தமிழ்' },
-  { locale: 'tr', name: 'Turkish', nativeName: 'Türkçe' },
-  { locale: 'uk', name: 'Ukrainian', nativeName: 'Українська' },
-  { locale: 'vi', name: 'Vietnamese', nativeName: 'Tiếng Việt' },
+  { tag: 'en', name: 'English', nativeName: 'English' },
+  { tag: 'ar', name: 'Arabic', nativeName: 'العربية' },
+  { tag: 'bg', name: 'Bulgarian', nativeName: 'Български' },
+  { tag: 'ca', name: 'Catalan', nativeName: 'Català' },
+  { tag: 'zh-Hans', name: 'Chinese (Simplified)', nativeName: '中文（简体）' },
+  { tag: 'zh-Hant', name: 'Chinese (Traditional)', nativeName: '中文 (繁体)' },
+  { tag: 'cs', name: 'Czech', nativeName: 'Čeština' },
+  { tag: 'da', name: 'Danish', nativeName: 'Dansk' },
+  { tag: 'nl', name: 'Dutch', nativeName: 'Nederlands' },
+  { tag: 'en-GB', name: 'English (U.K.)', nativeName: 'English (U.K.)' },
+  { tag: 'fi', name: 'Finnish', nativeName: 'Suomi' },
+  { tag: 'fr', name: 'French', nativeName: 'Français' },
+  { tag: 'gl', name: 'Galician', nativeName: 'Galego' },
+  { tag: 'de', name: 'German', nativeName: 'Deutsch' },
+  { tag: 'hi', name: 'Hindi', nativeName: 'हिन्दी' },
+  { tag: 'hu', name: 'Hungarian', nativeName: 'Magyar' },
+  { tag: 'id', name: 'Indonesian', nativeName: 'Bahasa Indonesia' },
+  { tag: 'it', name: 'Italian', nativeName: 'Italiano' },
+  { tag: 'ja', name: 'Japanese', nativeName: '日本語' },
+  { tag: 'ko', name: 'Korean', nativeName: '한국어' },
+  { tag: 'lt', name: 'Lithuanian', nativeName: 'Lietuvių' },
+  { tag: 'ml', name: 'Malayalam', nativeName: 'മലയാളം' },
+  { tag: 'fa', name: 'Persian', nativeName: 'فارسی' },
+  { tag: 'pl', name: 'Polish', nativeName: 'Polski' },
+  { tag: 'pt', name: 'Portuguese', nativeName: 'Português' },
+  { tag: 'pt-BR', name: 'Portuguese (Brazil)', nativeName: 'Português (Brasil)' },
+  { tag: 'pt-PT', name: 'Portuguese (Portugal)', nativeName: 'Português (Portugal)' },
```

Comment:
Hmm, something seems wrong when we have these three choices all next to each other.

It looks like the state of things:
https://www.transifex.com/zulip/zulip/mobile/
is that the pt-BR translation is quite fragmentary, but the pt and pt-PT translations are much more complete: 11% vs 89% and 64%. So I think the simplest reasonable thing to do for now is to leave the fragmentary one out of it. I'd hope that the pt translation is actually Brazilian Portuguese.

Some previous discussion here of how we might straighten out this area of our translations:
https://chat.zulip.org/#narrow/stream/58-translation/topic/language.20cleanup